lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -915,47 +916,48 @@ llvm::Error Caffe2ModelLoader::loadWeight(const caffe2::OperatorDef &op) {
      i: 127
      }
      */
-    auto *T = new Tensor();
     for (auto &o : op.output()) {
+      auto *T = new Tensor();
```

Comment:
Part of the point of making these into unique pointers would be so that if we return early due to an error, the tensor won't be leaked if it wasn't first put into `tensors_`. To that end, I think it would be better if we used make_unique() instead of keeping raw pointers then putting these into unique_ptrs with `reset`

Reply:
std::make_unique might not be available in all compilers we use.

Reply:
but I see your point, I'll send PR

Reply:
https://github.com/pytorch/glow/pull/2136